src_c/_freetype.c

```diff
@@ -2205,10 +2205,10 @@ MODINIT_DEFINE(_freetype)
     FREETYPE_MOD_STATE(module)->cache_size = 0;
     FREETYPE_MOD_STATE(module)->resolution = PGFT_DEFAULT_RESOLUTION;
 
-    Py_INCREF((PyObject *)&pgFont_Type);
+    Py_INCREF(&pgFont_Type);
     if (PyModule_AddObject(module, FONT_TYPE_NAME, (PyObject *)&pgFont_Type) ==
         -1) {
```

Comment:
This should just test the value directly, rather than comparing to `-1`, to be consistent with the other ones. Especially since `-1` on failure is not documented.

Reply:
oh, nice catch! :) I made this change in many other places but forgot to do it here